docs/source/interfaces/Engineering Diffraction Test Guide.rst

```diff
@@ -156,6 +156,44 @@ Negative Testing Ideas
 - Change any unlocked dialog boxes whilst `Fit` runs
 
 
+.. _gsas-Engineering-Diffraction_test-ref:
+
+GSAS Fitting
+^^^^^^^^^^^^
+
+GSAS-style fitting allows the user to plot peaks from their focused
+NeXuS file (obtained from
+:ref:`focus-Engineering_Diffraction_test-ref`).
+
+After a fit has run, the plot should look something like below:
+
+.. image:: ../images/EnggDiffExampleGSASOutput.png
+    :width: 500px
```

Comment:
Does RST support percentage specifiers here? For me I have to open the full image to actually read any of the text on it.